helm/aladdin-demo/templates/aladdin-demo.service.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Service
+metadata: 
+  name: {{ .Chart.Name }}
+  labels:
+    name: {{ .Chart.Name }}
+    app: {{ .Chart.Name }}
+spec:
+  # Aladdin will fill this in as NodePort which will expose itself to things outside of the cluster
+  # This is to highlight difference between public and private service types, and to only use
+  # public service types when it truly should be public
+  type: {{ .Values.service.publicServiceType }}
+  ports:
+    - name: http
+    # Expose the nginx port if we are using it, otherwise expose the uwsgi port
+{{ if .Values.app.nginx.use }}
+      port: {{ .Values.app.nginx.port }}
+{{ else }}
+      port: {{ .Values.app.port }}
+{{ end }}
+  selector:
+    # Get the aladdin-demo deployment configuration from aladdin-demo.deploy.yaml
+    deploymentconfig: {{ .Chart.Name }}
```

Comment:
we're getting rid of this label - for name as the selector. you can look at latest mc/websocket as reference